src/css/components/_progress.scss

```diff
@@ -27,6 +27,12 @@
 /* We need an increased hit area on hover */
 .video-js .vjs-progress-control:hover .vjs-progress-holder { font-size: 1.666666666666666666em }
 
+/* Also show the current time tooltip */
+.video-js .vjs-progress-control:hover .vjs-play-progress:after {
+  display: block;
+  font-size: 0.6em;
```

Comment:
If you set `.video-js .vjs-play-progress:after` to `display: block`, it would look fine at 0.9em.  If we don't reduce the font size here on hover, however, it ends up way too big (imo).